File: CiscoPCCE.Toolkit/Bean/InventoryFile.cs
using System.Xml.Serialization;

using CiscoPCCE.Toolkit.Sdk;

namespace CiscoPCCE.Toolkit.Bean
{
    [XmlRoot("inventoryFile")]
    public class InventoryFile : BaseApiBean
    {
        [XmlElement("content")]
        public string? Content { get; set; }

        [XmlElement("name")]
        public string? Name { get; set; }
    }
}

File: JavaSrc/toolkit/bean/InventoryFile.java
// ----------------------------------------------
package com.cisco.ccbu.cce.unifiedconfig.toolkit.bean;

import com.cisco.ccbu.cce.unifiedconfig.toolkit.BaseApiBean;
import javax.xml.bind.annotation.XmlSeeAlso;
import javax.ws.rs.Path;
import javax.xml.bind.annotation.XmlRootElement;
import java.util.List;
import com.cisco.ccbu.cce.unifiedconfig.toolkit.ReferenceBean;
import java.util.Date;
import com.cisco.ccbu.cce.unifiedconfig.toolkit.BaseApiListBean;

import javax.xml.bind.annotation.XmlElement;

import javax.xml.bind.annotation.XmlElementWrapper;


@XmlRootElement(name="inventoryFile")
public class InventoryFile extends BaseApiBean {
  private String content;
  private String name;

  public String getContent() {
     return this.content;
  }

  public void setContent(String content) {
     this.content = content;
  }

  public String getName() {
     return this.name;
  }

  public void setName(String name) {
     this.name = name;
  }


}
